SECURITY.md

```diff
@@ -0,0 +1,22 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+# Security Policy
+
+## Supported Versions
+
+We commit to publishing security updates for the last release of Slint:
```

Comment:
What about the rest?
The vscode extension, the viewer, ....


Maybe add something like "paid license holders may be eligible for support on additional versions, depending on their specific agreement terms."

Reply:
Good idea. Done.